app/workers/education_form/generate_spool_files.rb

```diff
@@ -114,6 +136,19 @@ def malformed_claim_ids
     end
     # :nocov:
 
+    def write_names_and_ssns
+      filename = Dir.mktmpdir + '/ssns.csv'
```

Comment:
I think we would want separate files for each region, since I believe they're going to send them to the people in each office to prioritize them.